sdk/storage/azblob/internal/generated/autorest.md

```diff
@@ -36,7 +36,7 @@ directive:
   transform: >-
     return $.
       replaceAll(`[]string{"2021-12-02"}`, `[]string{ServiceVersion}`).
```

Comment:
Does this need to be updated?

Reply:
Removed unnecessary line of code from here.